### PR TITLE
Fix broken and/or missing links in the documentation

### PR DESCRIPTION
**Pull request by ozturkberkay**
Hi,

 This PR fixes many broken links across the documentation. Sorry for the unnecessary amounts of commits, I did not expect such amount of broken links when I started; the list just kept getting bigger :)

**Note:** There are still many other broken links in the documentation. These are the ones I could fix.

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **cliveseldon**

If they are not already assigned, you can assign the PR to them by writing `/assign @cliveseldon` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Hi @ozturkberkay. Thanks for your PR.

I'm waiting for a [SeldonIO](https://github.com/orgs/SeldonIO/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/SeldonIO/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by cliveseldon** [comment]
Thanks @ozturkberkay this is really great.
Will add comments inline.

File: doc/source/python/python_wrapping_docker.md [hunk 1]
```diff
@@ -200,9 +200,9 @@ class MyModel(object):
     	return [{"type": "COUNTER", "key": "mycounter", "value": 1}]
 ```
 
-For more details on custom metrics and the format of the metric dict see [here](../custom_metrics.md).
+For more details on custom metrics and the format of the metric dict see [here](https://docs.seldon.io/projects/seldon-core/en/latest/analytics/analytics.html#custom-metrics).
```
Comment (cliveseldon) [inline doc/source/python/python_wrapping_docker.md:203]: Can we try to ensure all internal docs links remain relative as that ensures they will link to the same version when published wheras the above external link would move people to the latest version.

File: doc/source/python/python_wrapping_s2i.md [hunk 2]
```diff
@@ -249,9 +249,9 @@ class MyModel(object):
         return [{"type": "COUNTER", "key": "mycounter", "value": 1}]
 ```
 
-For more details on custom metrics and the format of the metric dict see [here](../custom_metrics.md).
+For more details on custom metrics and the format of the metric dict see [here](https://docs.seldon.io/projects/seldon-core/en/latest/analytics/analytics.html#custom-metrics).
```
Comment (RafalSkolasinski) [inline doc/source/python/python_wrapping_s2i.md:252]: I have the feeling we should try to keep relative links wherever possible

Comment (RafalSkolasinski) [reply doc/source/python/python_wrapping_s2i.md:252]: Somehow didn't see Clive's comment when browsing through changes. Sorry for duplicated!

**Comment by axsaucedo** [comment]
We decided to merge this and open #1851 to address the issue with relative links
/approve

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/SeldonIO/seldon-core/pull/1743#issuecomment-630940059" title="Approved">axsaucedo</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [axsaucedo]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
failed to trigger Pull Request pipeline
* failed to create agent
* failed to calculate in repo config
* failed to load trigger config for repository SeldonIO/seldon-core for ref 886692df64eafd6d4f0737909fbf73890b753a2e
* failed to find any lighthouse configuration files in repo SeldonIO/seldon-core at sha 886692df64eafd6d4f0737909fbf73890b753a2e
* failed to process repo SeldonIO/seldon-core refref 886692df64eafd6d4f0737909fbf73890b753a2e
* failed to list files in directory /var/tmp/gitrepo660600678/.lighthouse
* open /var/tmp/gitrepo660600678/.lighthouse
* no such file or directory

**Pull request merged**